generated/nisync/nisync.proto

```diff
@@ -200,6 +200,18 @@ enum NiSyncAttributes {
   NISYNC_ATTRIBUTE_8021AS_AS_CAPABLE = 1151115;
 }
 
```

Comment:
There are absolutely on enums in this file. Does nisync not have constants in header file? Or did they not port those constants as enums.

Reply:
I'm guessing they just haven't ported any constants over as enums. There's a handful of constants in the header file in the includes folder at least: [include/nisync.h](https://github.com/ni/grpc-device/blob/main/imports/include/nisync.h)